run_dir/static/js/data_del_plot.js

```diff
@@ -0,0 +1,349 @@
+window.current_plot_data = null;
+window.current_plot_obj = null;
+window.current_view_schemes = null;
+
+$(function(){
+    init_page_js();
+});
+
+function refresh_plot(){
+    var params = get_parameters();
+    if (window.current_plot_data == null){
+        get_plot_data(key=params[0], name=params[1], view_type=params[2], search_string=params[3], inst_type_filter=params[4]);
+    }else{
+        make_plot(key=params[0], name=params[1], view_type=params[2], inst_type_filter=params[4]);
+    }
+}
+
+function make_plot(key, name, view_type, filter_inst_type){
+
+  function formatBytes(bytes){
+      if (bytes === 0) return '0 Bytes';
+      const k = 1024;
+      const dm = 2 < 0 ? 0 : 2;
```

Comment:
I'm not sure I understand what this does? Isn't it just `const dm = 2`?

Reply:
hah, yes I think that `2` is likely to always be greater than `0`.. 😅 

Long hand version of this code:
```javascript
if(2 < 0){
  dm = 0;
} else {
  dm = 2;
}
```

Reply:
Ooops, this is a remnant from the function I found on Stackoverflow :-)